advanced.php

```diff
@@ -60,7 +60,7 @@
                      $rs->getCurrentValueByNr(0);
             $rs2 = $db->executeQuery($query);
             if($rs2->next())		// title found
-              echo $rs2->getCurrentValueByNr(0);
+              echo filter_var($rs2->getCurrentValueByNr(0),FILTER_SANITIZE_SPECIAL_CHARS);
```

Comment:
I think the sanitization should be done only at insertion. It feels strange to have to sanitize the db content :/

Reply:
You're right, that's a better way to do it. I was worried that txtDB would have issues dealing with special chars, but that's not the case. I'll sanitize them on insertion.